saleor/graphql/webhook/subscription_types.py

```diff
@@ -663,6 +664,19 @@ class Meta:
         interfaces = (Event,)
 
 
+class SaleToggle(ObjectType, SaleBase):
+    sale = graphene.Field(
+        "saleor.graphql.discount.types.Sale",
+        channel=graphene.String(
+            description="Slug of a channel for which the data should be returned."
+        ),
+        description="The sale the event relates to." + ADDED_IN_35 + PREVIEW_FEATURE,
+    )
+
+    class Meta:
```

Comment:
Hmm, it seems we are missing the general description of subscription events? We should have some description here that describes when the event is fired.

Reply:
I can add, but we are missing the description for all events types :D

Reply:
Let's add here for now and we'll open task to add for others separately.